docs/help-topics/stonesoup-import/stonesoup-import.yml

```
@@ -41,16 +41,23 @@
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - Before you expand all of your configuration options, check the following:
+        - Is the repository correct? Click the GitHub link to check your repository URL or click the repository name above the GitHub link to edit it. Click the checkmark to save any changes you make. To exit editing mode, click the X.
+        - **Runtime**: Did we detect the right one? If so, you should notice that we suggested appropriate deployment options. If you want to change your runtime, click the runtime name and make a selection.
+    - Expand the section next to your repository name, then expand **Build & deploy configuration** to look over these items:
+        - **Dockerfile path**: If we detected the wrong Dockerfile, click in the field to modify it.
+        - **Target port**: Is it correct? If not, click in the field to modify it.
+        - **CPU** and **Memory**: Choose how many of each of these you want for your application, depending on your deployment requirements.
+    - Expand **Show advanced deployment options**. 
+        - **Instances**: How many instances of your image do you want to use? The more instances, the more deployments to scale the availability of your application.
+        - **Environment variables**: Define your environment variables to customize how we deploy your application. If you defined your runtime environment variables in a devfile, we detect and extract them. 
+    - **Build pipeline**: Choose **Default** or **Custom** to specify how to trigger rebuilds.
```

Something like this might be more clear, but is less concise. I think it's helpful to do something to call out that the tasks will be different. 
```suggestion
    - **Build pipeline**: Choose **Default** or **Custom** to pick how builds are triggered and which tasks are run out of the box.
```